Chess-Challenge/src/My Bot/MyBot.cs

```diff
@@ -37,10 +37,14 @@ public Move Think(Board board, Timer timer)
             return bestMove;
         }
 
+        var rnd = new Random((int)DateTime.Now.Ticks);
+
         // TODO 2: Filter moves after which we are captured, or at least cheapest piece will be captured.
```

Comment:
Current code will filter out either all moves where our figure can be captured or none of them. We do not consider the approach "at least cheapest piece will be captured".

Reply:
@niklappo  correct. `At least cheapest piece` strategy is not straightforward, so I decided to not implemented it in this PR.